server.py

```diff
@@ -1,17 +1,38 @@
-from flask import Flask, request, jsonify, g
-from cryptography import exceptions
-from decrypter import Decrypter
-from structlog import wrap_logger
 import binascii
 import logging
 import os
 
+from cryptography import exceptions
+from flask import Flask, request, jsonify
+from sdc.crypto.decrypter import decrypt as sdc_decrypt
+from sdc.crypto.invalid_token_exception import InvalidTokenException
+from sdc.crypto.secrets import SecretStore, validate_required_secrets
 from sdx.common.logger_config import logger_initial_config
+from structlog import wrap_logger
+import yaml
+
+import settings
+
+EXPECTED_SECRETS = [
+    "SDX_SECRET_KEY",
+]
+
+KEY_PURPOSE_SUBMISSION = 'eq-submission'
 
 
 __version__ = "1.3.0"
 
+
 app = Flask(__name__)
+app.config.from_object(settings)
+
+app.sdx = {}
+
+with open(app.config['SDX_SECRETS_FILE']) as file:
```

Comment:
Seeing as we are changing this here, shall we jump in and make the app creation an app factory? This would help with unit testing etc http://flask.pocoo.org/docs/0.12/patterns/appfactories/